scripts/script.js

```diff
@@ -0,0 +1,77 @@
+/*eslint-disable */
```

Comment:
Please don't disable any linters. They are important for code standardization of your codebase.